Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 ﻿<Project>
```

Comment:
Does korebuild.json \ korebuild.lock.txt need to be updated too?

Reply:
korebuild-lock.txt was updated and I don't believe korebuild.json needs to be.